lighthouse-core/driver/gatherers/content-width.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const Gather = require('./gather');
+
+/* global document, window, __returnResults */
+
+/* istanbul ignore next */
+function getContentWidth() {
+  __returnResults({
+    scrollWidth: document.scrollingElement.scrollWidth,
+    viewportWidth: window.outerWidth
```

Comment:
PPK recommends diff properties for the "layoutViewport" and "visualViewport" on http://www.quirksmode.org/mobile/viewports2.html
Basically `document.documentElement.clientWidth` and `window.innerWidth`.  

Your scrollingElem.scrollwidth works for me. But how about the outerWidth; why use instead of innerWidth?

Whatever the answer, lets add as some inline comments. :)